Source/FieldSolver/WarpXPushFieldsHybridPIC.cpp

```diff
@@ -240,6 +241,10 @@ void WarpX::HybridPICDepositRhoAndJ ()
     // Perform current deposition at t_{n-1/2}.
     mypc->DepositCurrent(m_fields.get_mr_levels_alldirs(FieldType::current_fp, finest_level), dt[0], -0.5_rt * dt[0]);
 
+    // TODO: Perhaps add flag here for when using temperature accumulation in Hybrid
+    // Perform Temperature Deposition at time t_{n+1}
```

Comment:
Should this be? 
```suggestion
    // Perform Temperature Deposition at time t_{n}
```
Or is the comment on line 241 above wrong?

Reply:
I see the comment above the current deposition is wrong...